restdocs-api-spec-jsonschema/src/test/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGeneratorTest.kt

```diff
@@ -256,6 +315,14 @@ class JsonSchemaFromFieldDescriptorsGeneratorTest {
         )
     }
 
+    private fun givenFieldDescriptorWithRequiredArray() {
+        val notNullConstraint = Attributes(listOf(Constraint(NotNull::class.java.name, emptyMap())))
+        fieldDescriptors = listOf(
+            FieldDescriptor("obj", "some", "ARRAY", attributes = notNullConstraint),
```

Comment:
I suggest to rename this:
`obj` -> `array`

It's an array field in an object. The test assertion would then also need to be adapted.